src/ucp/core/ucp_request.c

```diff
@@ -338,11 +342,12 @@ void ucp_request_init_multi_proto(ucp_request_t *req,
     UCS_PROFILE_REQUEST_EVENT(req, multi_func_str, req->send.length);
 }
 
-static UCS_F_ALWAYS_INLINE void ucp_send_request_init_id(ucp_request_t *req)
+static UCS_F_ALWAYS_INLINE void
+ucp_send_request_proto_init_local_id(ucp_request_t *req)
```

Comment:
this is a common request init function, but it seems it checks a tag-protocol specific flag
need to use it only from tag send flow